src/test/java/com/baloise/confluence/digitalsignature/DigitalSignatureMacroTest.kt

```
@@ -0,0 +1,120 @@
+
+        Assertions.assertEquals(
+            "mailto:heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com,heinz.meier@meier.com?Subject=Subject",
+            mailto
+        )
+    }
+
+    @Test
+    fun mailtoVeryLong(): Unit {
+        val macro =
+            DigitalSignatureMacro()
+        val profiles: MutableList<UserProfile> = ArrayList()
+        val profile = buildUserProfile("Heinz Meier", "heinz.meier@meier.com")
+        for (i in 0..199) {
+            profiles.add(profile)
+        }
+
+        val mailto = macro.getMailto(profiles, "Subject", true, signature, "nirvana")
+
+        Assertions.assertEquals(
+            "nirvana/rest/signature/1.0/emails?key=signature.3224a4d6bba68cd0ece9b64252f8bf5677e24cf6b7c5f543e3176d419d34d517&signed=true",
+            mailto
+        )
+    }
+
+    @Test
+    fun mailtoShort(): Unit {
+        val macro = DigitalSignatureMacro()
+        val profiles: MutableList<UserProfile> = ArrayList()
+        val profile = buildUserProfile("Heinz Meier", "heinz.meier@meier.com")
+        profiles.add(profile)
+
+        val mailto = macro.getMailto(profiles, "Subject", true, null)
+
+        Assertions.assertEquals(
+            "mailto:Heinz Meier<heinz.meier@meier.com>?Subject=Subject",
+            mailto
+        )
+    }
+
+    fun buildUserProfile(userName: String, userEmail: String)= object : UserProfile {
+            override fun getFullName(): String = userName
+            override fun getEmail(): String = userEmail
+
+            override fun getUserKey(): UserKey {
+                TODO("Not yet implemented")
```

TODO will show up in tools like IntelliJ TODO list window.
Better to use `throw UnsupportedOperationException("Not yet implemented")`